.github/workflows/release.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: taiki-e/create-gh-release-action@v1
+      - uses: dtolnay/rust-toolchain@1.65
```

Comment:
this is the 'create-release' job, don't change this one

Reply:
oh whoops i think i autopiloted and deleted that by accident